### DISCUSSION TITLE
绘制组件参数polygonDrawingOpts.preRenderDatas支持异步数据吗

### DISCUSSION THREAD

**Discussion by ChauPha** [zouyaoji/vue-cesium#188] [Q&A] [answered]
polygonDrawingOpts.preRenderDatas参数支持异步数据吗 我用异步数据发现界面不
![1649657621(1)](https://user-images.githubusercontent.com/24471468/162675776-39b3f172-f1e7-4e3a-b475-8eb955c4de05.jpg)
渲染

**Comment by zouyaoji** [comment, marked as answer]
不支持。如果是异步的，只有加一个v-if条件，等异步完成之后v-if置为真
